bin/scrape_software_versions.py

```diff
@@ -7,43 +7,48 @@
 regexes = {
     'nf-core/metaboigniter': ['v_pipeline.txt', r"(\S+)"],
```

Comment:
This part could be extended with other software you are using OpenMS for instance or others